### ISSUE TITLE
attribute-hyphenation should apply to v-model:propName

### ISSUE THREAD

**Issue by catrope** [vuejs/eslint-plugin-vue#2510]
**What rule do you want to change?** attribute-hyphenation

**Does this change cause the rule to produce more or fewer warnings?** More

**How will the change be implemented? (New option, new default behavior, etc.)?** New default behavior

**Please provide some example code that this change will affect:**

```vue
<my-component
    v-model:selectedItem="..."
   defaultValue="..."
/>
```

**What does the rule currently do for this code?** It flags that the `defaultValue` prop is not hyphenated (should be `default-value`), but it does not flag the `selectedItem` prop.

**What will the rule do after it's changed?** In addition to complaining that `defaultValue` should be hyphenated (`default-value`), it will also complain that `selectedItem` should be hyphenated (`v-model:selected-item`)

**Additional context**
If prop names should be hyphenated, then they should always be hyphenated, even when used with `v-model`.

**Comment by waynzh** [comment]
Duplicate of https://github.com/vuejs/eslint-plugin-vue/issues/2439.

**Comment by catrope** [comment]
It's analogous to #2439 but it doesn't seem to be a duplicate? It's a similar issue, but in a different rule, and this aspect of it is not described in #2439 at all

**Comment by waynzh** [comment]
> in a different rule
You're right. Got confused there😬
